app/send-txs.hoon

```diff
@@ -100,13 +178,14 @@
 ++  wake-see
   |=  [wir=wire ~]
   ^-  [(list move) _this]
-  :_  this(see ~)
-  :_  ~
-  ^-  move
-  %+  batch-requests  /see
-  %+  turn  ~(tap in see)
+  =/  s  ~(tap in see)
+  =/  extra  (slag 50 s)
+  =/  check  (scag 50 s)
+  :_  this(see (silt extra))
+  %^  batch-requests  /see  |
+  %+  turn  check
   |=  txh=@ux
-  :-  `(crip '0' 'x' ((x-co:co 64) txh))
+  :-  `(crip 'see-' '0' 'x' ((x-co:co 64) txh))
```

Comment:
This works, but it feels wrong. I'd expect `(weld "see-0x" ((x-co:co 64) txh))` here.

Reply:
Sure